Chap 1/Chapter 1 CS/chapter1_1_home.txt

```diff
@@ -71,7 +71,7 @@ Swallowing hard, you consider the tactics you could employ as the reliable ack-a
 		A pale, familiar face stares right back at you.
 		*line_break
 		*line_break
-		Blood completely drained, jaws slackened in a horrified tableau. Blood drenches his neck, his body crumpled on the ground. He almost made it. Almost. Another soldier had been shot to pieces. He stood no chance against the LMG. [i]Fucking bastards![/i] You look around, your squad (or the remaining ${men} men) stand ready at the door leading into the base of the enemy building, clearly shaken.
+		Blood completely drained, jaws slackened in a horrified tableau. Blood drenches his neck, his body crumpled on the ground. He almost made it. Almost. Another soldier had been shot to pieces. He stood no chance against the LMG. [i]Fucking bastards![/i] You look around, only to see two more of your men collapsed on the ground, unmoving. Your squad (or the remaining ${men} men) stand ready at the door leading into the base of the enemy building, clearly shaken.
```

Comment:
the sentance sounds weird - You look around, only to see two more of your men collapsed on the ground, unmoving. Maybe can change unmoving to dead